Sources/Plasma/NucleusLib/pnInputCore/plKeyMap.h

```diff
@@ -54,8 +54,12 @@ You can contact Cyan Worlds, Inc. by email legal@cyan.com
 #include "plInputMap.h"
 #include "plControlEventCodes.h"
 
+#include <map>
+#include <string_theory/string>
 #include <vector>
 
+#include "plResMgr/plLocalization.h"
```

Comment:
NucleusLib should only include things from NucleusLib or CoreLib.

Reply:
yeah, but I think this is unavoidable without tackling the issue of localized key names, which is bigger in scope

It does suck that this gets publicly exposed in a header now 😕 (and who decided that localization was part of the resource manager? That makes no sense at all...)

Reply:
A forward declaration is probably sufficient at this time.

Reply:
Ah, hm. This is annoying. I think @dpogue's suggestion of converting this to a file-local `static` variable is the best solution probably, because then the header doesn't need to reference `plLocalization::Language` at all.

It doesn't solve the issue that pnInputCore links against plResMgr, but that was already the case before, so at least it's not getting any worse than before.